app/controllers/backend/fixed_assets_controller.rb

```diff
@@ -100,6 +100,7 @@ def show
 
       return unless @fixed_asset = find_and_check
       t3e @fixed_asset
+      notify_warning_now(:closed_financial_periods) if (@fixed_asset.started_on <= @fixed_asset.journal.closed_on) rescue false
```

Comment:
Could the check in the condition be replaced with a call to the method that is also checking that in order to have [DRY](https://en.wikibooks.org/wiki/Ruby_on_Rails/Getting_Started/Don%C2%B4t_repeat_yourself)-er code ?